code/modules/clothing/spacesuits/rig/modules/combat.dm

```diff
@@ -183,8 +183,12 @@
 		return 0
 
 	gun.Fire(target,holder.wearer)
+	on_shot()
```

Comment:
this seems redundant, what exactly does this proc add?

Reply:
This is used to update stasis charge immediately after you shot